modules/bookClass.js

```diff
@@ -0,0 +1,61 @@
+const titleInput = document.querySelector('#title');
+const authorInput = document.querySelector('#author');
+const formButton = document.querySelector('#submit-button');
+const bookList = document.querySelector('.book-display');
+
+export {
+  titleInput, authorInput, formButton, bookList,
+};
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.books = Book.getBooks() || [];
+  }
+
+  static getBooks() {
+    const bookInfo = localStorage.getItem('books');
+    if (bookInfo !== null && bookInfo !== undefined) {
+      return JSON.parse(bookInfo);
+    }
+    return [];
+  }
+
+  addBookToList() {
+    bookList.innerHTML = '';
+    if (Array.isArray(this.books)) {
+      this.books.forEach((book, i) => {
+        const bookItem = document.createElement('li');
+        bookItem.classList.add('book-list');
+        bookItem.innerHTML = `${book.title} by ${book.author}<button class="remove" data-index="${i}">Remove</button>`;
+        bookList.appendChild(bookItem);
+      });
+    }
+  }
+
+  removeBookFromList(i) {
+    this.books.splice(i, 1);
+    localStorage.setItem('books', JSON.stringify(this.books));
+    this.addBookToList();
+  }
+
+  handleFormSubmit() {
+    const title = titleInput.value;
+    const author = authorInput.value;
+    const book = new Book(title, author);
+    this.books.push(book);
+    localStorage.setItem('books', JSON.stringify(this.books));
+    this.addBookToList();
+  }
+
+  handleRemoveButtonClick(e) {
+    if (e.target.matches('.remove')) {
+      const {
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md#:~:text=app%20without%20errors.-,Refactor%20your%20functions%20to%20arrow%20functions.,-Use%20let%20and), Please use arrow syntax instead of regular syntax for these methods.